index.js

```diff
@@ -2,14 +2,80 @@ const Vis = require("./graphvis");
 const { Module, render } = require("./visRenderer");
 
 let Sequelize;
-const relationship = ({ source, target, associationType, as }) => {
-  const typeString = {
-    BelongsTo: `[arrowtail=crow, arrowhead=none, dir=both, arrowsize=0.60]`,
-    BelongsToMany: `[arrowtail=crow, arrowhead=crow, dir=both, arrowsize=0.60]`
-  }[associationType];
-  if (typeString) {
-    return `"${source.name}" -> "${target.name}" ${typeString}`;
-  }
+
+const relationships = (associations, arrowShapes = {}, arrowSize = 0.6) => {
+  const mappings = [];
+
+  const mapper = {
+    associationTypes: new Map([
+      ['BelongsToMany', {
+        typeArrowShapes: arrowShapes['BelongsToMany']
+          ? arrowShapes['BelongsToMany']
+          : ['none', 'crow'],
+        items: []
```

Comment:
I have a preference for storing the items in a different object. I feel like one object to store the rendering configuration, and another to store the model associations will be easier to understand

Reply:
I think that makes sense, especially since a later step just ends up rewriting the associations anyway. I think the idea for a separate object is the piece I was missing to simplify the deduplication process!

Reply:
I simplified this down to separate objects that handle a little better.